rs/consensus/src/consensus.rs

```diff
@@ -374,23 +374,23 @@ impl<T: ConsensusPool> ChangeSetProducer<T> for ConsensusImpl {
     /// There are two decisions that [ConsensusImpl] makes:
     ///
     /// 1. It must return immediately if one of the subcomponent returns a
-    /// non-empty [ChangeSet]. It is important that a [ChangeSet] is fully
-    /// applied to the pool or timer before another subcomponent uses
-    /// them, because each subcomponent expects to see full state in order to
-    /// make correct decisions on what to do next.
+    ///    non-empty [ChangeSet]. It is important that a [ChangeSet] is fully
+    ///    applied to the pool or timer before another subcomponent uses
+    ///    them, because each subcomponent expects to see full state in order to
+    ///    make correct decisions on what to do next.
```

Comment:
Was that done by the Rust formatter or clippy?

Reply:
Neither. It was done manually :') to satisfy a new lint introduced to clippy.

https://rust-lang.github.io/rust-clippy/master/index.html#/doc_lazy_continuation

Reply:
Ouch!